### DISCUSSION TITLE
Test Connection button for Nightscout

### DISCUSSION THREAD

**Discussion by JLChamberlain** [NightscoutFoundation/xDrip#2451] [Ideas]
If the user doesn't have any data to transfer to their Nightscout then it would be handy to have a 'Test Connection' button (like xDrip4iOS). Once the user has entered the data they can just press the 'Test Connection' button and see that the details they have entered are valid.

**Comment by Navid200** [comment]
Is this a request for xDrip in general?
Or, is it a request for Nightscout on Google Cloud?
Those two reside in two different repositories.  It helps to know exactly which one your request is related to?

**Reply by JLChamberlain** [reply]
For xDrip. As I said this feature is already in xDrip4iOS so I thought it would be in the normal xDrip but apparently not. I just think it would be a handy addition for people when they first add their Nightscout info in xDrip.

**Comment by Navid200** [comment]
I have no idea how this is done in the ios version.
But, it seems to me that it would require uploading a dummy value to Nightscouit.  Then, reading it back to confirm.  Then, deleting it.  But, I don't know what would happen if a real upload to Nightscout coincided with this.  

@tzachi-dar Any thoughts?
